File: php/ssl-certificates-chain-api.php
<?php

$apiUrl = 'https://api.threatintelligenceplatform.com/v1/sslCertificatesChain';

$params = array(
    'apiKey' => 'Your SSL Certificates Chain API key',
    'domainName' => 'threatintelligenceplatform.com'
);

$url = $apiUrl . '?' . http_build_query($params, '', '&');

print($url . PHP_EOL . PHP_EOL);

$response = file_get_contents($url);

$res = json_decode($response);

if (! $res)
    return;

$length = count($res);
if (! $length > 0)
    return;

echo 'Chain length: ' . $length . PHP_EOL;

File: ruby/ssl-certificates-chain-api.rb
require 'erb'
require 'json'
require 'net/https'
require 'uri'
require 'yaml'

api_key = 'Your SSL Certificates Chain API key'
domain = 'threatintelligenceplatform.com'

api_url = 'https://api.threatintelligenceplatform.com/v1/sslCertificatesChain'

url = api_url +
      '?apiKey=' + ERB::Util.url_encode(api_key) +
      '&domainName=' + ERB::Util.url_encode(domain)

begin
  buffer = Net::HTTP.get(URI.parse(url))
  json = JSON.parse(buffer)
  puts json.to_yaml
rescue Errno::ECONNRESET, Errno::EINVAL, EOFError, JSON::ParserError,
       Net::HTTPBadResponse, Net::HTTPHeaderSyntaxError, Net::ProtocolError,
       OpenSSL::SSL::SSLError, Timeout::Error, URI::InvalidURIError => e
  puts e
end